### PR TITLE
wr-224: Show latest 5 user articles in user profile

### PR DESCRIPTION
**Pull request by kovAnya**

### CODE REVIEW

File: frontend/src/pages/profile/components/user-latest-articles/libs/enums/own-articles-config.enum.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const ownArticlesConfig = {
```
Comment (andanf-e) [inline frontend/src/pages/profile/components/user-latest-articles/libs/enums/own-articles-config.enum.ts]:
```suggestion
const ownArticlesPaginationConfig = {
```

File: frontend/src/pages/profile/components/user-latest-articles/user-latest-articles.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    void dispatch(
+      articlesActions.fetchOwn({
+        take: ownArticlesConfig.TAKE,
+        skip: ownArticlesConfig.SKIP,
+      }),
+    );
+  }, [dispatch]);
+
+  useEffect(() => {
+    getLatest();
```
Comment (andanf-e) [inline frontend/src/pages/profile/components/user-latest-articles/user-latest-articles.tsx]:
```suggestion
  useEffect(() => {
    void dispatch(
      articlesActions.fetchOwn({
        take: ownArticlesConfig.TAKE,
        skip: ownArticlesConfig.SKIP,
      }),
    );
```

**Review by andanf-e — CHANGES_REQUESTED** [review]

**Review by andanf-e — DISMISSED** [review]

File: frontend/src/pages/profile/components/user-latest-articles/user-latest-articles.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      <ol>
+        {articles.map((article) => (
+          <li key={article.id} className={styles.articleTitle}>
+            <Link
+              to={
+                AppRoute.ARTICLES_$ID.replace(
+                  ':id',
+                  article.id.toString(),
+                ) as typeof AppRoute.ARTICLES_$ID
+              }
```
Comment (andanf-e) [inline frontend/src/pages/profile/components/user-latest-articles/user-latest-articles.tsx]:
```suggestion
              to={configureString(
                  AppRoute.ARTICLES_$ID,
                  { id: String(article.id) }
                ) as AppRoute.ARTICLES_$ID
              }
```

File: frontend/src/pages/profile/components/user-latest-articles/user-latest-articles.tsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+              className={styles.articleLink}
+            >
+              {article.title}
+            </Link>
+          </li>
+        ))}
+      </ol>
+      <Link
+        to={AppRoute.ARTICLES_MY_ARTICLES as typeof AppRoute.ARTICLES_$ID}
```
Comment (andanf-e) [inline frontend/src/pages/profile/components/user-latest-articles/user-latest-articles.tsx]: I'm not sure that you need to cast one AppRoute enum to another
```suggestion
        to={AppRoute.ARTICLES_MY_ARTICLES}
```

**Review by andanf-e — CHANGES_REQUESTED** [review]

**Review by andanf-e — APPROVED** [review]

**Review by andanf-e — COMMENTED** [review]
Please also provide screenshot

**Comment by kovAnya** [comment]
> Please also provide screenshot

image

**Review by andanf-e — DISMISSED** [review]

**Review by andanf-e — COMMENTED** [review]
Maybe let's also add heading like `Your latest articles` to this section

**Review by mhrechkin — DISMISSED** [review]

**Comment by kovAnya** [comment]
image

**Review by andanf-e — APPROVED** [review]

**Review by NazarKovtun1 — APPROVED** [review]

**Pull request merged**